Framework/Core/src/WorkflowHelpers.cxx

```diff
@@ -203,6 +203,15 @@ void WorkflowHelpers::addMissingOutputsToBuilder(std::vector<InputSpec> const& r
   }
 }
 
+// get the default value for condition-backend
+std::string defaultConditionBackend()
+{
+  if (getenv("DDS_SESSION_ID") != nullptr || getenv("OCC_CONTROL_PORT") != nullptr) {
+    return "o2-ccdb.internal";
```

Comment:
I think it should actually be `http://o2-ccdb.internal`
Can you also add an env variable that I can use to override it, while the command line parsing is not working?